packages/app/src/components/DBRowSidePanel.tsx

```diff
@@ -158,7 +158,9 @@ export default function DBRowSidePanel({
   }
 
   const mainContentColumn = getEventBody(source);
-  const mainContent: string | undefined = normalizedRow?.['__hdx_body'];
+  const mainContent: string | undefined = normalizedRow?.['__hdx_body']
+    ? JSON.stringify(normalizedRow['__hdx_body'])
```

Comment:
style: I know we are using these aliases on the frontend. we should create an enum for them. 

Another issue here is the `JSON.stringify` generates incorrect results on a string, right? (like adding extra quotes). I think we want to do this for the object type only

Reply:
Yeah, I am also hesitate to `JSON.stringify` everything under `normalizedRow['__hdx_body']`.

The reason I am doing this is when select `Implicit Column Expression` return type is not `string`, it most likely will get error(number, object, null, [code ref](https://github.com/hyperdxio/hyperdx/blob/2cf978978558b96a6c4bdf740b9cb7be60800ccb/packages/app/src/components/DBRowSidePanelHeader.tsx#L54)) or not display correctly([1,2,3] become 123).

But you are correct, `JSON.stringify` will create double quote on string. I will make stringify only work when the input is not string.